win-odbc64/odbcclient/update_version.pl

```diff
@@ -97,4 +97,80 @@ sub update_file {
     update_file $file
 }
 
+# update the version in build_os.bat
+sub update_build_os {
+    my $infile = $_[0];
+    print  "Update " , $infile, "\n";
+    my $outfile = $infile + '.tmp';
+    open( INFILE, $infile ) or die "Error: Can't open $infile - $!";
+    open( OUTFILE, ">$outfile" ) or die "Error: Can't open $outfile - $!";
+
+    while ( <INFILE> ) {
+        if (/TFODBC64-/) {
+            s/(\d+)\.(\d+)\.(\d+)/$major.$minor.$sp/;
+            print OUTFILE;
+        }
+        else {
+            print OUTFILE;
+        }
+    }
+
+
+    close( INFILE ) or warn "Warning: Can't close $infile - $!";
+    close( OUTFILE ) or warn "Warning: Can't close $outfile - $!";
+
+    unless ( rename $outfile, $infile ) {
```

Comment:
In my environment, new file will has the same permissions as the old. However the developer who use the tool to update the version should check the new file again before committing them. 
Since we have too many files need to be updated in windows odbc driver when we update the Traofion version, it's very easy to miss some file if we don't use a tool.